qemu/tests/cfg/format_disk.cfg

```diff
@@ -13,7 +13,9 @@
     format_cmd = "cd /dev && for i in `ls | egrep [shv]db`;do yes |mkfs.ext3 $i; done"
     list_disk_cmd = ""
     set_online_cmd = ""
+    show_dev_cmd = "cd /dev && ls | egrep [shv]db"
     mount_cmd =  "mkdir -p /media && cd /dev && ls | egrep [shv]db | xargs -I dev mount -t ext3 dev /media"
+    show_mount_cmd = "mount | grep -Eo '/dev/[shv]db'"
```

Comment:
@CongSmile Double check the code and maybe the code of format disk need some update. So it is OK to still separated the cmds like this. Let's use other pullreqs to fix this problem. And thanks for your explain.